# origin: Debian (herbert)
# cset: n/a
# inclusion: submitted (2004-09-18)
# description: don't build cpqphp_proc.o if !PROC_FS
# revision date: 2004-09-05

diff -urN kernel-source-2.4.26/drivers/hotplug/Makefile kernel-source-2.4.26-1/drivers/hotplug/Makefile
--- kernel-source-2.4.26/drivers/hotplug/Makefile	2003-08-25 21:44:41.000000000 +1000
+++ kernel-source-2.4.26-1/drivers/hotplug/Makefile	2004-04-17 08:17:42.000000000 +1000
@@ -18,7 +18,6 @@
 
 cpqphp-objs		:=	cpqphp_core.o	\
 				cpqphp_ctrl.o	\
-				cpqphp_proc.o	\
 				cpqphp_pci.o
 
 ibmphp-objs		:=	ibmphp_core.o	\
@@ -36,5 +35,9 @@
 	cpqphp-objs += cpqphp_nvram.o
 endif
 
+ifeq ($(CONFIG_PROC_FS),y)
+	cpqphp-objs += cpqphp_proc.o
+endif
+
 include $(TOPDIR)/Rules.make
 
